understanding/21/non-text-contrast.html

```diff
@@ -396,6 +396,19 @@ <h4>Required for Understanding</h4>
 					<li> The information is available in another form, such as in a table that follows the graph, which becomes visible when a "Long Description" button is pressed.</li>
 					<li> The graphic is part of a logo or brand name (which is considered &quot;essential&quot; to its presentation).</li>
 				</ul>
+
+				<div class="note">
+					<p>Logos and logotypes are exempted from contrast requirements when they are purely used as
+						<em>graphical objects</em>, under the assumption that they must comply with stricter color choices mandated
+            by corporate identity guidelines.
+						However, this is not the case when they also act as <em>user interface components</em>
+						(such as links or other interactive controls). In these cases, authors should choose
+						a variant of the logo or logotype that has sufficient contrast, if allowed by the
+						corporate identity guidelines.
+						Alternatively, authors should provide an equivalent <em>user interface component</em>
+						which serves the same purpose and does meet the contrast requirements.</p>
```

Comment:
```suggestion
					<p>Logos and logotypes are exempted from contrast requirements when they are purely used as <em>graphical objects</em>, under the assumption that they must comply with stricter color choices mandated by corporate identity guidelines. They are not excempt when they also act as <em>user interface components</em> (such as links or other interactive controls). 
            When logos and logotypes are used as <em>user interface components</em>, authors should choose a variant of the logo or logotype that has sufficient contrast, if allowed by the  corporate identity guidelines. Alternatively, authors should provide an equivalent <em>user interface component</em> which serves the same purpose and does meet the contrast requirements.</p>
```

I understand the value and meaning of should in RFCs. I hope this slight rephrasing and reformatting makes the text more explicit and understandable.

Reply:
Visually, the code looks like you want that update as two paragraphs, but there's only one set of `<p>` tags. Should it be one or two?

Reply:
I'm assuming the salient bit was the addition of this in the first para:

`They are not exempt when they also act as <em>user interface components</em> (such as links or other interactive controls).`

(with correction of `excempt` to `exempt` as well)

Reply:
Noting that making this categorical statement will no doubt ruffle some feathers, as it "seems" to introduce a normative failure by the backdoor that some will argue wasn't as cut and dry there before...which is why I was treading lightly (in one of the linked issues there was some mention of "exempting logos was on purpose as it would lead to too many failures" or words to that effect)

Reply:
happy to take it as additional suggestion to the WCAG 2.x backlog group though for further discussion

Reply:
@erikkroes I updated your suggestion above for clarity (keeping the line breaks where they were, so it's clearer what your change is), and assuming you did want the first para to be actually split into two as well

Reply:
I've opened this issue for clarification, so maybe I tread less lightly 😄

The split might not be needed indeed. I think it helped make my point in this conversation, but it should be fine without. 
Thanks for picking this up and taking it further!